src/test/java/com/google/songgame/TitleFormatterTest.java

```diff
@@ -0,0 +1,128 @@
+package com.google.songgame.data;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/*
+  https://music.youtube.com/playlist?list=RDCLAK5uy_kmPRjHDECIcuVwnKsx2Ng7fyNgFKWNJFs
+
```

Comment:
Not sure if this extra whitespace matters

Reply:
+1, let's remove the extra line.

In addition, could you add a description of what this playlist means, e.g. "Test cases taken from this playlist"